gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -235,7 +231,7 @@ private void scheduleSpecsFromCatalog() {
   private void clearRunningFlowState(Iterator<URI> drUris) {
     while (drUris.hasNext()) {
       // TODO: Instead of simply call onDeleteSpec, a callback when FlowSpec is deleted from FlowCatalog, should also kill Azkaban Flow from AzkabanSpecProducer.
-      onDeleteSpec(drUris.next(), FlowSpec.Builder.DEFAULT_VERSION);
+      unscheduleSpec(drUris.next(), FlowSpec.Builder.DEFAULT_VERSION);
```

Comment:
We're not using this DR code anyway, but based on comments it seems like it's intended that it kills running flows that are marked as DR, so maybe let's leave this method as is.